snakemake/script.py

```diff
@@ -393,7 +393,7 @@ def evaluate(self, edit=False):
 
     @property
     def local_path(self):
-        path = self.path[7:]
+        path = str(self.path)#[7:]
```

Comment:
Here, I'd rather rely on the SourceFile methods for checking whether the path is absolute or not and so on.